package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "jest",
+    "testwatch": "jest --watchAll",
+    "watch": "webpack --watch",
+    "dev": "webpack --mode development",
+    "start": "webpack serve --mode development --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/marcode95/to-do-list.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/marcode95/to-do-list/issues"
+  },
+  "homepage": "https://github.com/marcode95/to-do-list#readme",
+  "devDependencies": {
+    "babel-eslint": "10.1.x",
+    "css-loader": "^5.2.0",
+    "eslint": "6.8.x",
+    "eslint-config-airbnb-base": "14.1.x",
+    "eslint-plugin-import": "2.20.x",
+    "style-loader": "^2.0.0",
+    "stylelint": "13.3.x",
+    "stylelint-config-standard": "20.0.x",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "3.17.x",
+    "webpack": "^5.30.0",
+    "webpack-cli": "^4.6.0"
+  },
+  "dependencies": {
+    "aos": "^2.3.4",
+    "webpack-dev-server": "^3.11.2"
+  }
```

Comment:
- [ ] Kindly add the` webpack-dev-server` to the dependencies. This will ensure that proper configuration is done for your application.  This should be done as it is requested In the project description ⬇️ 
![devdep_requirement](https://user-images.githubusercontent.com/32361963/114868426-91e6ad00-9dfe-11eb-84ac-0cad9a0ec714.png)

Reply:
-bug preventing users from editing/deleting to-do's on the project and future pages fixed
-adding multiple projects is already working for me, what kind of bug is it? please clarify in the next review
-since it is saying 'some sort of' I took the liberty to just create a starting page showing all the to-do's which I think is a simpler solution, users can access the starting page by click on the logo in the upper-left corner
-title added to pull request
-redirect after submitting a project added